### PR TITLE
Remove react-highlight.js

### PR DESCRIPTION
**Pull request by dogmar**
[Linear ticket](https://linear.app/pluralsh/issue/ENG-578/consolidate-highlightjs-and-react-highlightjs-into-one-import)
Prevents double-import of `highlight.js` library

- Removed all references to `react-highlight.js`
- Adjusted existing code just enough to prevent TS and linting errors
- Updated Notifications panel layout to use Markdown component from design system

### CODE REVIEW

File: www/src/components/incidents/Markdown.js [hunk 1]
```diff
@@ -4,13 +4,16 @@ import {
...
-import hljs from 'highlight.js'
+import {
+  Copy, TooltipContent, WithCopy,
+} from 'forge-core'
+import hljs from 'highlight.js/lib/core'
 import { normalizeColor } from 'grommet/utils'
 import sortBy from 'lodash/sortBy'
 import { Emoji as Emojii } from 'emoji-mart'
 
+import Highlight from '../utils/Highlight'
```
Comment (maciaszczykm) [inline www/src/components/incidents/Markdown.js:15]: We can export this to design system as the next step as it looks like part of Markdown.

**Review by maciaszczykm — APPROVED** [review]
Looks good! I've restarted backend test as it seems to be random failure.

File: www/src/components/incidents/Markdown.js (hunk 1 above)
Comment (floreks) [reply www/src/components/incidents/Markdown.js:15]: We should create an issue to not forget about it.

Comment (maciaszczykm) [reply www/src/components/incidents/Markdown.js:15]: I will take care of it.

Comment (maciaszczykm) [reply www/src/components/incidents/Markdown.js:15]: https://linear.app/pluralsh/issue/ENG-584/export-highlight-component-to-design-system

**Comment by maciaszczykm** [comment]
I'll merge to finish replacing Highlight component by the one from design system.

**Pull request merged**